apps/i18n/common/id_id.json

```diff
@@ -812,9 +815,9 @@
   "minecraftAquaticButton": "Pergi ke pendidikan Minecraft",
   "minecraftAquaticDesc": "Lanjutkan pembelajaran dengan Minecraft.",
   "minecraftAquaticTitle": "Pendidikan Minecraft",
-  "minecraftMarketingButton": "Mulai",
-  "minecraftMarketingDesc": "",
-  "minecraftMarketingTitle": "Minecraft: Perjalanan Pahlawan",
+  "minecraftAquaticPromoButton": "Mulai",
+  "minecraftAquaticPromoDesc": "",
+  "minecraftAquaticPromoTitle": "",
```

Comment:
Are these strings the same and we just changed the keys? Or were they for an older mc tutorial? Because if they were the same, we're losing translations, no?

Reply:
it looks like "minecraftMarketingButton" is the same and was preserved, but the titles are different (the old one was "Hero's Journey"). So we did lose the translations for that string, but it looks like we're not using it anymore.